library/std/src/io/buffered/tests.rs

```diff
@@ -844,7 +844,7 @@ fn long_line_flushed() {
 }
 
 /// Test that, given a very long partial line *after* successfully
-/// flushing a complete line, that that line is buffered unconditionally,
+/// flushing a complete line, that line is buffered unconditionally,
```

Comment:
There are a couple of targets for `that` / `that line` -- it could be `a very long partial line` or `a complete line`.

It's hard to be certain (a) which line is intended and (b) which line is being referenced.

The best approach is probably more prose and less `that`. I'm happy to write it, but I'm not sure what the intent is.

I suspect `that the very long partial line` is intended.